tic_tac_toe/game.py

```diff
@@ -87,18 +130,47 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
 
+    if game['winner'] or _board_is_full(game['board']):
+        raise InvalidMovement("Game is over.")
+    
+    elif get_next_turn(game) != player:
+        raise InvalidMovement('"{}" moves next'.format(game['next_turn']))
+    if _position_is_valid(position):
+        if not _position_is_empty_in_board(position, game['board']):
+            raise InvalidMovement("Position already taken.")
+        game['board'][position[0]][position[1]] = player
+        if game['next_turn'] == game['player1']:
+            game['next_turn'] = game['player2']
+        else:
+            game['next_turn'] = game['player1']
+    else:
+        raise InvalidMovement("Position out of range.")
+    
+    if _check_winning_combinations(game['board'], player):
+        game['winner'] = player
+        game['next_turn'] = None
+        raise GameOver('"{}" wins!'.format(player)) 
+    elif _board_is_full(game['board']) and not game['winner']:
+        game['winner'] = None
+        game['next_turn'] = None
+        raise GameOver('Game is tied!')
+        
 
 def get_board_as_string(game):
     """
     Returns a string representation of the game board in the current state.
     """
-    pass
+    first_line = '  |  '.join(game['board'][0])
+    second_line = '  |  '.join(game['board'][1])
+    third_line = '  |  '.join(game['board'][2])
+    separater = '\n--------------\n'
+    string_board = '\n' + first_line + separater + second_line + separater + third_line + '\n'
+    return string_board
 
```

Comment:
Interesting way to do it! I think a loop could simplify a little bit. But your code is pretty clever!